src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,122 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import jdk.nashorn.internal.objects.annotations.Constructor;
+
+@Entity
+public class Book {
+
+    @Column()
+    private String genre;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private int pages;
+
+    @Column(nullable = false)
```

Comment:
In this attribute (isbn) you can put the following parameter in the @Column annotation to indicate that it is unique.
```suggestion
    @Column(nullable = false, unique = true)
```